src/Ozone_Monitor/data_xtrct/ush/OznMon_MkBase.sh

```diff
@@ -199,28 +146,25 @@ for type in ${SATYPE}; do
     
    while [[ $cdate -le $EDATE ]]; do
 
-      pdy=`echo $cdate | cut -c1-8 `
-      cyc=`echo $cdate | cut -c9-10`
-
-      test_dir=${OZN_STATS_TANKDIR}/${RUN}.${pdy}/${cyc}/oznmon/time
+      test_dir=$(${MON_USH}/get_stats_path.sh --run $RUN --pdate ${cdate} \
```

Comment:
Change `$RUN` to `${RUN}` for consistency.

Reply:
Yup, and thanks.  I've been trying to do that in every file I touch.